### PR TITLE
Logging improvements

### PR DESCRIPTION
**Pull request by vvp**
This PR introduces few minor improvements on logging. Potentially user-space breaking change is that `RUST_LOG` environment variable now overrides the `--verbosity` setting (rationale is that `RUST_LOG` allows more fine-grained log level setting for debug/development purposes).

### CODE REVIEW

**Review by ljedrz — APPROVED** [review]
LGTM, though I'd squash these changes into a single commit for easier rebases later on

File: cli/src/helpers/logger.rs [hunk 1]
```diff
@@ -12,25 +12,31 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use crate::helpers::LogWriter;
+use std::{env::VarError, fs::File, io, path::Path};
```
Comment (ljedrz) [inline cli/src/helpers/logger.rs]: `clippy` complains that `env::VarError` is unused

File: node/src/validator/mod.rs [hunk 2]
```diff
@@ -258,8 +258,9 @@ impl<N: Network, C: ConsensusStorage<N>> Validator<N, C> {
 
         // Load the primary's keys.
         let primary_key_file = format!("{base_path}.primary-{primary_id}-key.json");
-        let primary_keypair =
-            read_authority_keypair_from_file(primary_key_file).expect("Failed to load the node's primary keypair");
+        let primary_keypair = read_authority_keypair_from_file(primary_key_file).unwrap_or_else(|_| {
+            panic!("Failed to load the node's primary keypair {base_path}.primary-{primary_id}-key.json")
```
Comment (joske) [inline node/src/validator/mod.rs]: why not add the actual error?

Comment (vvp) [reply node/src/validator/mod.rs]: Oh good point, I was using `except(format!(...))` before and just blindly took clippy's advice for this replacement. Will add the error logging too.

**Pull request merged**